### PR TITLE
Quick fix for cutoff title

### PR DESCRIPTION
**Pull request by Gaurav0**
This fixes the following issue on canary:

screen shot 2015-08-04 at 4 36 06 pm

### CODE REVIEW

**Comment by miguelcobain** [comment]
This always annoyed me.

:+1:

**Comment by joostdevries** [comment]
does not fix it for me on dev

File: app/styles/app.scss [hunk 1]
```diff
@@ -98,6 +98,7 @@ body {
 		  border: 0;
 		  color: #faf2ee;
 			cursor: pointer;
+      box-sizing: content-box;
```
Comment (joostdevries) [inline app/styles/app.scss]: you need to apply this on the `input` element itself, not the wrapper.

**Comment by Gaurav0** [comment]
@joostdevries Seems that somewhere you have a * { box-sizing: border-box } on canary that I don't have on my local copy of master. So it inherited for me, but not for you. Why don't we have the same styles?

Anyway, fixed.

File: app/styles/app.scss [hunk 2]
```diff
@@ -103,6 +103,7 @@ body {
 				background-color: transparent;
 				border: 0;
 				outline: 0;
+        box-sizing: content-box;
```
Comment (joostdevries) [inline app/styles/app.scss]: what's up with the indent?

Comment (Gaurav0) [inline app/styles/app.scss]: tabs vs spaces. the usual.

Comment (joostdevries) [inline app/styles/app.scss]: maybe convert the entire file to spaces to avoid messiness.

Comment (Gaurav0) [inline app/styles/app.scss]: Done.

**Comment by joostdevries** [comment]
:+1:

**Pull request merged**